pkg/cover/backend/backend.go

```diff
@@ -65,7 +65,7 @@ type Range struct {
 
 const LineEnd = 1 << 30
 
-func Make(target *targets.Target, vm, objDir, srcDir, buildDir string, splitBuild bool,
+func Make(target *targets.Target, vm, objDir, srcDir, buildDir string, splitBuild, isKernel61OrEarlier bool,
```

Comment:
This new argument looks a bit too specific when compared to all others that don't depend on any OS or platform.

`targets.Target` could theoretically encapsulate it. Or, rather, for the `NrFtracePlts` const, which is the only thing affected by `isKernel61OrEarlier` in the end (or did I miss something?).

But the problem is that there's no good way to change `targets.Target` argument dynamically (there's a `SYZ_CLANG` hack, but that's not something to take inspiration from). And the `targets.Target` is also quite generic, `NrFtracePlts` would also look kind of out of place there.

Maybe return to the discussed idea of querying it from the vmlinux file? `strings vmlinux-4f5e5092 | grep "Linux version"` does seem to work, but maybe there's something cleaner.